molotov/tests/test_fmwk.py

```diff
@@ -115,10 +123,16 @@ def test_runner_multiprocess(self):
         args = self.get_args()
         args.processes = 2
         args.workers = 5
+        args.console = console
         results = runner(args)
         self.assertTrue(results['OK'] > 0)
         self.assertEqual(results['FAILED'], 0)
 
+    @dedicatedloop
+    def test_runner_multiprocess_console(self):
+        self._multiprocess(console=True)
```

Comment:
console is the flag that determine if Molotov runs with the console or the curses output (-c in the shell)